src/deluge/util/container/array/ordered_resizeable_array_with_multi_word_key.cpp

```diff
@@ -119,3 +119,29 @@ bool OrderedResizeableArrayWithMultiWordKey::deleteAtKeyMultiWord(uint32_t* __re
 		return false;
 	}
 }
+
+void OrderedResizeableArrayWithMultiWordKey::testSequentiality(char const* errorCode) {
```

Comment:
Can we share the code with `ResizeableArray` somehow instead of duplicating it?

Reply:
Not without a restructure to make ResizeableArray inherit from ResizeableArrayWithMultiWordKey instead of the other way around - technically this code would work for single word keys as well, but the single word version doesn't work for multi words